README.md

```diff
@@ -44,7 +44,7 @@ Installing/Building
 
 ### K Backends
 
-There are three backends of K available: LLVM (default) for concrete execution and Haskell (default) and Java for symbolic execution.
+There are two backends of K available: LLVM for concrete execution and Haskell for symbolic execution.
```

Comment:
Perhaps we should remove OpenJDK from the dependencies list as well; I can do that in #1558 .

Reply:
OpenJDK is needed needed for K itself.